plugins/tech-radar/src/utils/components.tsx

```diff
@@ -14,22 +14,29 @@
  * limitations under the License.
  */
 import React from 'react';
+import { Link } from '@backstage/core-components';
 
 type WithLinkProps = {
   url?: string;
+  urlTarget?: string;
   className: string;
   children: React.ReactNode;
 };
 
 export const WithLink = ({
   url,
+  urlTarget,
   className,
   children,
 }: WithLinkProps): JSX.Element =>
-  url ? (
-    <a href={url} className={className}>
```

Comment:
Hmm. I'd actually vote for skipping the new field and passing of target entirely and just make target blank the default unless there are good reasons for not making that the default.

Reply:
I'd say don't use target at all. Let the Link decide what to do. It decides what to do based on whether it's an internal or external link, and in the future the Link may even be tied to per user preferences on what preferred target behavior they have. Just leave it unset :)

Reply:
Even better!

Reply:
Let me do some testing!!!